test/models/GuidelineSpecifiedDocumentTest.scala

```diff
@@ -0,0 +1,13 @@
+package models
+
+import org.scalatest.FunSuite
+
+class GuidelineSpecifiedDocumentTest extends FunSuite {
+  val guidelineId = "ER3-2558"
+
+  test("GuidelineSpecifiedDocument") {
```

Comment:
ชื่อ test ไม่ได้บอกจุดประสงค์ของการ test เลย ซึ่งอย่าง case นี้เทสว่า "GuidelineSpecifiedDocument is constructed with guideLineId" อะไรประมาณนี้ หรือแล้วแต่จะตั้ง naming guideline กันยังไง

ประเด็นคือ อันนี้มัน test constructor และ getter (ประมาณนั้น) ซึ่งผมว่ามันไม่จำเป็น แล้วถ้ามีเทสแบบนี้เยอะๆ แล้วมันไม่ได้ protect business value อะไร มันจะกลายเป็น code ที่ไม่ add value แล้วทำให้ maintain ลำบากขึ้นเปล่าๆ ครับ เอาไปเทสตอนมีคนเอามันไปใช้ดีกว่า อันนี้มัน too fine-grianed

Reply:
ยอมรับว่าไม่ได้คิดตรงนี้เลยครับ ตอนทำ model เห็นกลุ่มอื่นมี test model กันเลยเข้าใจว่าต้องทำ test ตรงนี้  ขอบคุณสำหรับคำแนะนำครับ